lib/sass/script/functions.rb

```diff
@@ -1670,6 +1687,42 @@ def feature_exists(feature)
     end
     declare :feature_exists, [:feature]
 
+    # Returns a reference to a function for later invocation with the `call` function.
+    #
+    # The function reference created may refer to a function defined in
+    # your stylesheet, built-in to the host environment, or a CSS native
+    # function. As such, this function never returns an error, but the
+    # you can use `function-exists()` to check if the function reference
+    # created points at a Sass function.
```

Comment:
It think I'd prefer this not support CSS functions. The fact that function calls can be interpreted as plain CSS already makes typos very painful to debug; I don't think we should recapitulate that where it's not necessary.

Maybe we can add an optional `$css` function to control whether it's a plain-css function, with the default being `false`.